### ISSUE TITLE
Windows Implementation

### ISSUE THREAD

**Issue by ghost** [Fmstrat/youtube-dl-api#3]

**Comment by Fmstrat** [comment]
That's the URL you can reach the service at. So, let's say you're not using SSL and just doing this on a local network. Compose would look like this:

```
  youtube-dl-api:
    image: nowsci/youtube-dl-api
    container_name: youtube-dl-api
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /storage/video/Watch Later:/data
    ports:
      - 8080:8080
    environment:
      - PORT=8080
      - TOKEN=<yourtoken>
      - EXTHOST=http://192.168.0.111:8080
      - FORMAT=%(title)s - %(uploader)s - %(id)s.%(ext)s
    restart: always
```

Then, you would visit `http://192.168.0.111:8080?token=<mytoken>`. In this case, your windows machine has the IP 192.168.0.111.

The `ports` section says forward port 8080 from the host to 8080 on the container. The `environment` section sets environment variables inside the container (think things like PATH). Those variables are then used by the python script inside it.

That help?

**Comment by Scalisee** [comment]
I appreciate the help. That definitely makes sense, as I'm not looking to remote. Per your feedback I plugged my local IP address and opened the port, but I'm still getting an issue with the volumes string. I pointed the path to a local file on my I: drive, clearly I'm still missing something. There following is my current .YML file and the subsequent error I receive (<hidden>):

youtube-dl-api:
    image: nowsci/youtube-dl-api
    container_name: youtube-dl-api
    volumes:
      - I:\Videos\Watch Later
    ports:
      - 8080:8080
    environment:
      - PORT=8080
      - TOKEN=<token>
      - EXTHOST=http://<local IP>
      - FORMAT=%(title)s - %(uploader)s - %(id)s.%(ext)s
    restart: always

ERROR: for youtube-dl-api  Cannot create container for service youtube-dl-api: b'Mount denied:\nThe source path "9af8bdb649ead46a301de9bc8a45f6adea652c6595e9c04b9ff4313b1e319edf:I:\\\\Videos\\\\Watch Later"\nis not a valid Windows path'
ERROR: Encountered errors while bringing up the project.
